en/editor/publish/build-guide.md

```diff
@@ -79,118 +80,119 @@ It is divided into the following steps:
 
-All used UUIDs will be sorted during building, and the ones that appear **twice or more** will be stored in the `uuids` array, and the position of the previously used UUID will be replaced with the index. All `types` that appear **twice or more** will also be stored in the `types` array, and the previously used position will be replaced with the index.
+The `UUIDs` used in the `Bundle` will be sorted during the build, and those that appear **twice or more** will be stored in the `uuids` array, and the location of the previously used `UUID` will be replaced with the index.
 
-#### Build resources
+All `types` that appear **twice and more** are also stored in the `types` array, and the previously used position is replaced with the index.
 
-The building of resources in this step is to package resource files other than scripts, because scripts are packaged separately as special files. In the stage of packaging resources, the editor will first summarize **currently participating in the building of the scene and resources in all Bundles directory. The packaging of each resource will be deserialized by the engine, and the dependent resources will be found and then recursively packaged**. Before deserialization, the script environment of the entire project will be configured, that is, all non-plug-in project scripts will be loaded. Since the loading of the script will directly affect the deserialization, if the script is not legally loaded, it will directly cause the build to fail. If a dependent resource is found to be lost during the deserialization process, a warning will be issued, but the building will continue. The warning here does not mean that the problem does not need to be resolved. **If the resource is lost and not resolved, it is difficult to ensure that the content after the build is free of problems**.
-
-After the resource is deserialized during the packaging process, it will be recompressed and serialized to reduce the package body after packaging. The serialized files of texture resources will all be packaged into a JSON file, and other serialized files will be subpackaged according to the build configuration parameters.
-
-#### Build script
+#### Build resources
 
-The script building in the editor is divided into **plug-in scripts** and **non-plug-in scripts**. For details, please refer to the description of **script building** in the previous section.
+This step generates project resource files other than scripts, as scripts are compiled and processed separately as special files. The resources are re-serialized after deserialization is performed during the packaging process to reduce the package size after packaging. Multiple serialized JSONs are combined into a single file based on the JSON grouping information collated from the previous data, e.g. the serialized files of the `texture` resource are all packaged into a single `json` file.
```

Comment:
```suggestion
This step generates project resource files other than scripts, since scripts are compiled and processed separately as special files. The resources are re-serialized after deserialization is performed during the packaging process to reduce the package size after packaging. Multiple serialized JSONs are combined into a single file based on the JSON grouping information collated from the previous data, e.g. the serialized files of the `texture` resource are all packaged into a single JSON file.
```